autoload/maktaba/value.vim

```diff
@@ -228,6 +228,19 @@ function! maktaba#value#IsEnum(Value) abort
 endfunction
 
 
+""
+" Coerce {Value_or_values} into a list, wrapping any non-list value into a
```

Comment:
nit: lowercase `value_or_values` in documentation I think? (That's consistent with what we have elsewhere in this file, though also I'm not sure why we have initial caps for the function parameter names anyway.)

Reply:
It's capitalized because it can technically be called with a funcref and vim weirdly requires the variables those are stored in to be capitalized (`:help E707`).

I thought vimdoc required the name to exactly match, but maybe that doesn't apply to initial caps. I'll try.

Reply:
Yeah, apparently doesn't need the name to match, no warnings for that or any other vimdoc blocks doing the same in this file. Updated.